NTO/Automation/attributes/actionDefinitions.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix ogit.Automation:        <http://www.purl.org/ogit/Automation/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Automation:actionDefinitions
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "actionDefinitions";
+	dcterms:description "Defines a json map of action handlers with capabilities, applicabilities.";
```

Comment:
We should try to avoid json encoded blobs in the model if possible, the least we should have here is that the value list has the capability as key and the applicability as value, although I wonder if we could somehow have it "deconstructed" even more... but probably with the applicability value being a map again our possibilities are limited